hack/chains/kaniko-demo.sh

```diff
@@ -20,11 +20,33 @@ oc project tekton-chains
 
 title "Ensure we have the demo kaniko build task"
 kubectl apply -f https://raw.githubusercontent.com/tektoncd/chains/main/examples/kaniko/kaniko.yaml
-
-# Tweak it to avoid TLS failures
-# Fixme: Make it work without --skip-tls-verify
-kubectl patch task/kaniko-chains --type=json \
-  --patch='[{"op": "add", "path": "/spec/steps/1/args/5", "value":"--skip-tls-verify=true"}]'
+# Patch the above task to add the volume with CA certificates from the chains-ca-cert secret and to add the volume mount
+# Go will try several paths to load the CA certificates, the key in the secret is 'ca-certificates.crt' and it matches
+# the /etc/ssl/certs/ca-certificates.crt path (see https://go.dev/src/crypto/x509/root_linux.go)
+kubectl patch task kaniko-chains --type=json --patch='[
+  {
+    "op": "add",
+    "path": "/spec/volumes",
```

Comment:
I don't understand what that means or what the pros and cons are, but this is consistent with components/build/tekton-chains/chains-controller-cert.yaml , so maybe that's fine. Also I'd say let's not over-think it for a demo.

Reply:
I think a workspace might be better for an optional volume or one that could be overridden when starting the TaskRun. So that `tkn task start...` below would need to point to the `chains-ca-cert` secret instead. Honestly given the demo purpose of this I'm not seeing a benefit to that.